An update in the ReadMe.md to be more specific with the element composition function. (#58)
grimme-lab/MindlessGen: CHANGELOG.md

- vdW radii scaling parameter can now be adjusted via `mindlessgen.toml` or CLI
- The check_distance function now checks based on the sum of the van der Waals radii and a scaling factor acessible via `mindlessgen.toml` or CLI
- better type hints for `Callables`

> - A more clear difference between the different scaling factors for the van der Waals radii.

Reviewer: ```suggestion
- A clearer differentiation between the distinct scaling factors for the van der Waals radii.
```

> - Update in the `ReadMe.md` which explains more detaild the element composition function.

Reviewer: ```suggestion
- `README.md` with more detailed explanation of the element composition function.
```
